Implement support for key revision.tag=last (#25)
dynamist/sgit: docs/revision-tag.md

# Revision: tag

If you want to use the more specialized and advance option for parsing and selecting a specific tag from a git repo then you should use the tag revision option.

This is the following syntax and extra options. More detailed examples is further down in this document.

```yaml
repos:
  pykwalify:
    clone-url: git@github.com:Grokzen/pykwalify.git
    revision:
      tag: latest

      # Pre-processes the tags before matching and removes any that we do not want
      # This step runs before clean step.
      # Multiple regex values is supported but it will stop parsing at first match and remove it
      tag-filter-regex:
        - "([0-9].[0-9].[0-9])"

      # Pre-processes the tags before selection step. Create a regex with a group that you want
      # to extract the semver data from. Multiple regex values is supported but first value matching
      # will be used before moving on to next tag.

>       # This step is usefull to remove and prefix or suffixes from tags and only care about the semver part when comparing tags

Reviewer: ```suggestion
      # This step is useful to remove any prefix or suffix from tags, only affecting the semver part when comparing tags
```
